lib/katello/version.rb

```diff
@@ -1,3 +1,3 @@
 module Katello
-  VERSION = "4.3.0.rc4".freeze
+  VERSION = "4.3.0".freeze
```

Comment:
You may want to rebase so that this commit is last, that way the tag will be on the version bump commit.